Meadowlark.net/Meadowlark.Net.Core/Backend/Utility.cs

```diff
@@ -0,0 +1,23 @@
+namespace Meadowlark.Net.Backend.Postgresql;
+
+public static class Utility
+{
+
+  /**
+   * Simple only-once guard for a function with no parameters
+   */
+  public static Action Once(Action fn){
+    var called = false;
```

Comment:
We'll want to [lock this](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/lock) for thread safety.